README.md

```diff
@@ -3,18 +3,94 @@
 A collection of Maven plugins and utilities that can be used to build CICS bundles, ready to be installed into CICS TS.
 
 This project contains:
- - `cics-bundle-maven-plugin`, a Maven plugin that builds CICS bundles, including Java-based dependencies.
- - `cics-bundle-reactor-archetype`, a Maven archetype that shows how a simple reactor build can pull together a CICS bundle and a Java project.
+ - `cics-bundle-maven-plugin`, a Maven plugin that builds CICS bundles, in which you can include any supported bundlepart.
```

Comment:
I think this is better than what's there, but maybe we should update it to something like:

A Maven plugin that authors CICS bundles, for deploying resources into CICS TS.  Supports a subset of bundle parts, including Java assets.